packages/radix-ui-themes/src/components/card.css

```diff
@@ -139,7 +139,18 @@
 
   &:where(button, [href]) {
     @media (hover: hover) {
-      &:where(:hover)::after {
+      &:where(:hover) {
+        &::after {
+          box-shadow: 0 0 0 1px var(--gray-a7);
+
+          @supports (box-shadow: 0 0 0 1px color-mix(in oklab, white, black)) {
+            box-shadow: 0 0 0 1px color-mix(in oklab, var(--gray-a7), var(--gray-7) 25%);
+          }
+        }
+      }
+    }
+    &:where([data-state='open']) {
+      &::after {
```

Comment:
Added missing [data-state='open'] styles as we usually pair them with all :hover styles